membership-attribute-service/app/controllers/AccountController.scala

```diff
@@ -254,9 +206,6 @@ class AccountController(
         paymentDetails <- OptionEither.liftOption(tp.paymentService.paymentDetails(\/.fromEither(freeOrPaidSub)).map(Right(_)).recover { case x =>
           Left(s"error retrieving payment details for subscription: ${sub.name}. Reason: $x")
         })
-        upToDatePaymentDetails <- OptionEither.liftOption(getUpToDatePaymentDetailsFromStripe(sub.accountId, paymentDetails).map(Right(_)).recover {
-          case x => Left(s"error getting up-to-date card details for payment method of account: ${sub.accountId}. Reason: $x")
-        })
```

Comment:
the stripe information is synced to zuora by the stripe-source-updated lambda in support-service-lambdas?
That is not 100% reliable so the extra check would provide some value.  Although perhaps the extra check was introduced before the lambda?

Are you saying that it's reliable enough compared with the impact of getting it wrong in mma, that the extra cost of the call is not worth it?  If so I probably agree.  But I just want to confirm the logic used here.

Reply:
I believe the check was put in place before the lambda existed (@paulbrown1982 could confirm). Yes to your second paragraph, the performance cost is not worth the benefit of picking up any inconsistencies.